docs/config/listeners.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Listeners
+---
+
+{% capture overview %}
+Listeners define how Honeytrap listens to the incoming traffic. This can be directly on sockets, raw or using the indirectly using the agent listener.
```

Comment:
Maybe "Listeners define how Honeytrap listens to the incoming traffic. This can be directly on sockets, raw or through agents as indirect listeners."?